app/coffee/Features/Cooldown/CooldownManager.coffee

```diff
@@ -0,0 +1,24 @@
+RedisWrapper = require('../../infrastructure/RedisWrapper')
+rclient = RedisWrapper.client('cooldown')
```

Comment:
I've lost touch with how we do this, are we setting a different connection group for each feature?